staff/dtiburcio/leadtech-site-semantic-exercise/index.html

```diff
@@ -0,0 +1,317 @@
+        </tr>
+        <tr>
+            <td><a href="https://www.workable.com/j/C00A7BBC5E">PHP or Java Developer PCI DSS</a></td>
+            <td>Spain, Catalonia, Barcelona</td>
+        </tr>
+        <tr>
+            <td><a href="https://www.workable.com/j/A22AC95D6D">PHP Senior Developer</a></td>
+            <td>Spain, Catalonia, Barcelona</td>
+        </tr>
+        <tr>
+            <td><a href="https://www.workable.com/j/54B0053D27">QA Automation Engineer</a></td>
+            <td>Spain, Catalonia, Barcelona</td>
+        </tr>
+        <tr>
+            <td><a href="https://www.workable.com/j/02B6AD3D91">QA Tester</a></td>
+            <td>Spain, Catalonia, Barcelona</td>
+        </tr>
+        <tr>
+            <td><a href="https://www.workable.com/j/5343E42174">Quality Specialist</a></td>
+            <td>Spain, Catalonia, Barcelona</td>
+        </tr>
+        <tr>
+            <td><a href="https://www.workable.com/j/9F0E7CC0D5">SEM Specialist</a></td>
+            <td>Spain, Catalonia, Barcelona</td>
+        </tr>
+        <tr>
+            <td><a href="https://www.workable.com/j/EEE6C4A347">Senior Java Developer</a></td>
+            <td>Spain, Catalonia, Barcelona</td>
+        </tr>
+        <tr>
+            <td><a href="https://www.workable.com/j/49087088B7">SEO Specialist</a></td>
+            <td>Spain, Catalonia, Barcelona</td>
+        </tr>
+        <tr>
+            <td><a href="https://www.workable.com/j/3744B8F64D">SEO Team Lead</a></td>
+            <td>Spain, Catalonia, Barcelona</td>
+        </tr>
+        <tr>
+            <td><a href="https://www.workable.com/j/2994CAD980">Tax & Accounting Specialist</a></td>
+            <td>Spain, Catalonia, Barcelona</td>
+        </tr>
+        <tr>
+            <td><a href="https://www.workable.com/j/715C823430">UX Designer</a></td>
+            <td>Spain, Catalonia, Barcelona</td>
+        </tr>
+    </table>
+</section>
+
+<section>
+    <h1>down to business</h1>
+    <h3>leadtech provides optimized solutions and services for all kinds of
+        online projects.</h3>
+    <section>
+        <h3>for Entrepreneurs</h3>
+        <p>
+            Feeling innovative? leadtech, with its pioneering and progressive
+            mindset, is just the place for fresh and forward-thinking proposals.
+        </p>
+        <button><a href="https://leadtech.com/entrepreneur">Show us your ideas</a></button>
+    </section>
+    <section>
+        <h3>for Investors</h3>
+        <p>
+            leadtech is taking investors into the future with daily successes, rapid
+            growth and consistently fruitful developments.
+        </p>
+        <button><a href="https://leadtech.com/investor">Let’s see your strategy</a></button>
+    </section>
+</section> 
+
+<section>
+    <h2>Stay tuned with our community</h2>
+    <h3>with our community</h3>
+    <a href="https://business.facebook.com/leadtech.social/?business_id=426848764180365"><img src="img/facebook.png"></a>
```

Comment:
This should be a list and icons are not semantic here (should be added through CSS)